docs/_static/credential/v1.yaml

```diff
@@ -0,0 +1,246 @@
+                type: string
+components:
+  schemas:
+    CredentialSubject:
+      type: object
+      description: Contents of a Verifiable Credential giving name and DID of organization.
+    KeyValuePair:
+      type: object
+      description: used search params
+      required:
+        - key
+        - value
+      properties:
+        key:
+          description: >
+            Fields from VCs to search on. Concept specific keys must be prepended with the concept name and a '.'.
+            Default fields: issuer, subject, type
+          type: string
+          example: company.name
+        value:
+          type: string
+    SearchRequest:
+      type: object
+      description: >
+        Input for a search call. Parameters are entered as key/value pairs.
+        Concept specific query params need to be prepended with the concept name.
+      required:
+        - params
+      properties:
+        params:
+          description: key/value pairs
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValuePair'
+        offset:
+          description: skips first x results, default 0
+          type: number
+        limit:
+          description: limit number of return values to x, default 10
+          type: number
+    VerifiableCredential:
+      type: object
+      description: A credential according to the W3C spec.
+      required:
+        - context
+        - type
+        - issuer
+        - issuanceDate
+        - credentialSubject
+        - proof
+      properties:
+        context:
+          description: List of URIs
+          type: array
+          items:
+            type: string
+        id:
+          description: credential ID. A Nuts DID followed by a large number.
+          example: "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY"
+          type: string
+        type:
+          description: List of type definitions for the credential. Always includes 'VerifiableCredential'
+          type: array
+          items:
+            type: string
+        issuer:
+          description: DID according to Nuts specification
+          example: "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY"
+          type: string
+        issuanceDate:
+          description: rfc3339 time string when the credential was issued.
+          type: string
+          example: "2012-01-02T12:00:00Z"
+        expirationDate:
+          description: rfc3339 time string untill when the credential is valid.
+          type: string
+          example: "2012-01-02T12:00:00Z"
+        credentialSubject:
+          $ref: '#/components/schemas/CredentialSubject'
+        proof:
+          description: one or multiple cryptographic proofs
+          type: object
+    VerificationOutcome:
+      required:
+        - status
+      properties:
+        status:
+          description: if the credential is valid, revoked, untrusted or has some other status.
+          type: boolean
+        statusReason:
```

Comment:
maybe rename it to `invalidReason` or something, because `statusReason` it might contain a value for when the status is valid.

Reply:
removed it for now since delete returns 201